src/components/ComparisonContainer/index.tsx

```diff
@@ -0,0 +1,504 @@
+          <div className="my-schedules">
+            <p className="content-title">My Schedules</p>
+            {allVersionNames.map((version) => {
+              return (
+                <ScheduleRow
+                  key={version.id}
+                  id={version.id}
+                  type="Version"
+                  onClick={(): void => handleToggleSchedule(version.id)}
+                  checkboxColor={selected.includes(version.id) ? '#FFFFFF' : ''}
+                  name={version.name}
+                  // placeholder functions
+                  handleEditSchedule={(): void => {
+                    setEditInfo({
+                      id: version.id,
+                      type: 'Version',
+                    });
+                    setEditValue(version.name);
+                  }}
+                  handleRemoveSchedule={(): void => {
+                    setDeleteConfirm({
+                      id: version.id,
+                      type: 'Version',
+                      name: version.name,
+                    });
+                  }}
+                  hasDelete={allVersionNames.length >= 2}
+                  editOnChange={(
+                    e: React.ChangeEvent<HTMLInputElement>
+                  ): void => setEditValue(e.target.value)}
+                  editOnKeyDown={handleEdit}
+                  editInfo={editInfo}
+                  editValue={editValue}
+                />
+              );
+            })}
+          </div>
+          <div className="shared-schedules">
+            <p className="content-title">Shared with me</p>
+            {sharedSchedules.map((friend) => {
+              return (
+                <div key={friend.id} className="friend">
+                  <ScheduleRow
+                    // change id later on
+                    id={friend.id}
+                    type="User"
+                    hasCheck={false}
+                    name={friend.name}
+                    handleEditSchedule={(): void => {
+                      setEditInfo({
+                        id: friend.id,
+                        type: 'User',
+                      });
+                      setEditValue(friend.name);
+                    }}
+                    handleRemoveSchedule={(): void => {
+                      setDeleteConfirm({
+                        id: friend.id,
+                        type: 'User',
+                        name: friend.name,
+                      });
+                    }}
+                    hasTooltip
+                    editOnChange={(
+                      e: React.ChangeEvent<HTMLInputElement>
+                    ): void => setEditValue(e.target.value)}
+                    editOnKeyDown={handleEdit}
+                    editInfo={editInfo}
+                    editValue={editValue}
+                  />
+                  {friend.schedules.map((schedule) => {
+                    return (
+                      <ScheduleRow
```

Comment:
I think we decided not to let schedules' names be customizable for now since it might confuse users that they'll be changing the actual names of their friends' schedules.